localization/english/dvg_flavor_lotharingia_l_english.yml

```diff
@@ -0,0 +1,106 @@
+﻿l_english:

```

Comment:
[07:23:34][pdx_localize.cpp:279]: Duplicate localization key. Key 'modifier_army_dissaproves' is defined in both 'localization/english/dvg_flavor_lotharingia_l_english.yml' and 'localization/english/dvg_modifiers_l_english.yml'.
[07:23:34][pdx_localize.cpp:279]: Duplicate localization key. Key 'Lysander' is defined in both 'localization/english/dvg_names_l_english.yml' and 'localization/english/names_l_english.yml'.